menas/src/test/scala/za/co/absa/enceladus/menas/integration/repositories/BaseRepositoryTest.scala

```diff
@@ -29,7 +29,7 @@ import scala.concurrent.{Await, Future}
 
 abstract class BaseRepositoryTest extends WordSpec with TestContextManagement with BeforeAndAfter {
 
-  val awaitDuration: Duration = Duration(500, TimeUnit.MILLISECONDS)
+  val awaitDuration: Duration = Duration(800, TimeUnit.MILLISECONDS)
```

Comment:
The embedded mongo take a little extra time, so sometimes 500ms was not enough.

Reply:
`[2020-04-08T10:03:56.432Z]   java.util.concurrent.TimeoutException: Futures timed out after [800 milliseconds]`

Might this be why the build is failing?

Reply:
Yes, apparently it seems to be machine-dependent.

Reply:
I am talking with @hamc17 . It also took 23 minutes which is too long. Let's see if he finds anything.

Reply:
The next build on Jenkins took about 17 minutes, seems about normal to me. But sure, @hamc17's consultation couldn't hurt 👍

Reply:
@Zejnilovic was there any thoughts? Should I contact @hamc17 directly on the topic?

Reply:
No, it's fine.